app/consumer.py

```diff
@@ -23,6 +24,32 @@ def get_delivery_count_from_properties(properties):
     return delivery_count + 1
 
 
+def _get_value(key):
+    value = os.getenv(key)
+    if not value:
+        raise ValueError("No value set for " + key)
+
+
+def check_default_env_vars():
+
+    env_vars = ["RECEIPT_HOST", "RECEIPT_PATH", "RECEIPT_USER",
+                "RECEIPT_PASS", "RABBITMQ_QUARANTINE_QUEUE", "RECEIPT_RRM_QUEUE", "RABBITMQ_EXCHANGE",
+                "SDX_RECEIPT_RRM_SECRET", "RABBITMQ_HOST", "RABBITMQ_HOST2", "RABBITMQ_PORT",
+                "RABBITMQ_PORT2", "RABBITMQ_DEFAULT_USER", "RABBITMQ_DEFAULT_PASS", "RABBITMQ_DEFAULT_VHOST"]
+
+    missing_env_var = False
+
+    for i in env_vars:
```

Comment:
Might be cleaner to do the check in the block and list the env var as a separate value to the log:
```python
for var in env_vars:
    if not os.getenv(var):
        logger.error("Failed to start", error="Missing env var", var=var)
        missing_env_var = True
```